spec/index.html

```diff
@@ -1867,9 +1980,12 @@ <h3>Plain Literal Event</h3>
 
       <p class="note">RDF/XML plain literals are in RDF 1.2 treated as
```

Comment:
"plain literals" went away with RDF 1.1.

We can define "plain literal" for the purposes of RDF/XML as "no given rdf:datatype".
It does not have to be a full `<dfn>`.

Digression 😀 
It may be useful to define "string literal" in RDF Concepts with alt "RDF string literal".

[SPARQL has one](https://www.w3.org/TR/sparql12-query/#func-string) because the grouping of xsd:string, rdf:langString and now rdf:dirLangString comes up repeated.

Reply:
I updated the language, although it may not flow quite as well. Suggestions are welcome.